tests/coalaTest.py

```diff
@@ -34,7 +34,7 @@ def test_coala(self):
             self.assertIn('This file has 1 lines.',
                           stdout,
                           'The output should report count as 1 lines')
-            self.assertIn('During execution of coala', stderr)
+            # self.assertIn('During execution of coala', stderr)
```

Comment:
As already mentioned in https://github.com/coala/coala/pull/3338#discussion_r111240835 and further explained in https://github.com/coala/coala/pull/3338#issuecomment-293686866 , this is obsolete :)